util/validation.ttl

```diff
@@ -0,0 +1,73 @@
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+
+@prefix valid: <http://industrialdataspace.org/2017/01/ids/validation#> .
+
+# Description of this ontology
+# ----------------------------
+
+valid: a owl:Ontology;
+    dct:title "The Industrial Dataspace Information Model Validation Ontology"@en;
+    rdfs:comment "An ontology to annotate RDF predicates with information needed to generate an API for the Information Model.";
```

Comment:
the name "The Industrial Dataspace Information Model Validation Ontology" is imho strongly misleading with regard to what it is - can we rename this?

Reply:
As these properties are currently relevant for transforming the ontology into a java-based representation (suitable for developers to easily create instances), we could rename it to "IDS Transformation Hints Ontology". Would that be more clear?